### DISCUSSION TITLE
Compass Rose <or> Direction-of-Travel?

### DISCUSSION THREAD

**Discussion by TriggerDingus** [organicmaps/organicmaps#5001] [Q&A]
I come from the days of hardcopy maps as a driver, SCUBA diver and hiker. The common thread is I always needed to know my direction of travel for orientation and route finding. I know this isn't a requirement for use of digital navigation, but I have an innate need to quickly and easily understand my directions.

The early versions of google maps had a compass rose on the map during navigation, a feature which they soon eliminated because eventually younger folks never had to learn magnetic directions at all. I degoogled myself many years ago and have been using a Garmin GPS and though they don't have a compass rose option, they do offer a way to enable 'direction of travel' in the status bar so I'm able to quickly determine direction that way.

I've been trying to learn and get used to Organic Maps and I've not found an option to denote direction-of-travel during navigation. I noticed I can press the blue arrow while running a route, but that rotates the map so North is upwards and so that isn't helpful.

Am I missing something obvious? Is there a compass rose or direction-of-travel feature I'm missing?

**Comment by biodranik** [comment]
When the map is rotated in the direction of the device (this mode is changed when you press a location button under + and -), a compass icon is displayed at the top right, with a red arrow pointing to the north. Did you see it?

**Comment by TriggerDingus** [comment]
Thanks for the reply! It's great there may actually be a compass in the app, but apparently I'm not seeing it on my device. Attached, I've enclosed a screen cap of a route navigation - one is before tapping the blue arrow and the other is after (which rotates the screen so north is upward). I don't see the compass rose anywhere, though.
![Screenshot_20230418-105221](https://user-images.githubusercontent.com/75340371/232818730-538f1ac6-200f-474f-8b55-77a41b94e3c8.png)
![Screenshot_20230418-105214](https://user-images.githubusercontent.com/75340371/232818744-9ccbdeb4-2ee6-494a-b9b9-34dad5584c1d.png)

**Reply by biodranik** [reply]
You're right, I forgot that during navigation the compass is not drawn. Can you please create an issue for it? And the existing compass may not be the best solution here, we likely need something from cars, like rotating W/S/E/N letters, or something better. There was an issue about improving the navigation UI, this element should be taken into an account for a redesign.

**Comment by TriggerDingus** [comment]
Issue created
#5010
